variants/vmware-dev/Cargo.toml

```diff
@@ -33,6 +32,7 @@ included-packages = [
     "release",
     "kernel-6.1",
     "open-vm-tools",
+    "netdog-vmware-config",
```

Comment:
this package doesn't exist anymore, should be removed

Reply:
oop you are right